sql/moz-fx-data-shared-prod/legal_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Legal tables
+description: |-
+  Reference data
+dataset_base_acl: derived
+user_facing: false
+labels: {}
+default_table_workgroup_access:
```

Comment:
You don't need to specify `default_table_workgroup_access`, as it is generally derived from `workgroup_access`.